Charon/filetypes/GCodeFile.py

```diff
@@ -68,9 +69,13 @@ def parseHeader(stream: IO[bytes], *, prefix: str = "") -> Dict[str, Any]:
 
             flavor = metadata.get("flavor", None)
             if flavor == "Griffin":
-                if metadata["header_version"] != "0.1":
+                if metadata["header_version"] != "0.1" and not SkipHeaderValidation:
```

Comment:
Minor remark, I think this read more natural
```suggestion
                if not SkipHeaderValidation and metadata["header_version"] != "0.1":
```